src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -205,6 +205,23 @@ public FeedbackSessionAttributes copyFeedbackSession(String newFeedbackSessionNa
 
         return viewableSessions;
     }
+    
+    /**
+     * checks if there is any email sent from the course
+     * @param courseId
+     * @param userEmail
+     * @return true if there is any email sent from the course.
```

Comment:
Update the javadoc as well. It is no longer **any** email, and the `userEmail` param is not used.